[Email Security] Inline deployment docs (#19099)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/email-security/setup/pre-delivery-deployment/inline-deployment.mdx

---
title: Inline deployment
pcx_content_type: concept
sidebar:
  order: 2
---

With pre-delivery deployment, also known as Inline deployment, Email Security evaluates email messages before they reach a user's inbox.

Email Security becomes a hop in the SMTP processing chain and physically interacts with incoming email messages. Based on your policies, various messages are blocked before reaching the inbox.

When you choose an inline deployment, you get the following benefits:

> - Messages are processed and physically blocked before delivery to a user's mailbox.

Reviewer: ```suggestion
- Messages are processed and physically blocked before arriving in a user's mailbox.
```
